.github/workflows/sdk-regression.yml

```diff
@@ -31,9 +31,31 @@ jobs:
           - percy-appium-js
           - gatsby-plugin-percy
     steps:
+      - name: Get user permissions
```

Comment:
isnt this file editable by user in a pr ?

Reply:
Yes but the run would use the workflow file from master unless it is merged